ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/cveDetailPage.test.ts

```diff
@@ -0,0 +1,73 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+
+describe('Node CVEs - CVE Detail Page', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_VULN_MGMT_NODE_PLATFORM_CVES')) {
+            this.skip();
+        }
+    });
+
+    it('should restrict access to users with insufficient permissions', () => {
+        // check that users without Node access cannot access the Node CVE Detail page directly
+    });
+
+    it('should only show relevant filters for the Node CVE Detail page', () => {
+        // check the advanced filters and ensure only the relevant filters are displayed
+    });
+
+    it('should link to the correct pages', () => {
+        // clicking the Node CVEs breadcrumb should navigate to the overview page with the CVE tab selected
+        // clicking a Node name in the list should navigate to the correct Node details page
+    });
+
+    it('should sort Node table columns', () => {
+        // check sorting of Node column
+        // check sorting of CVE Severity column
+        // check sorting of CVE status column
+        // check sorting of CVSS column
+        // check sorting of cluster column
+        // check sorting of Operating System column
+        // check sorting of Affected components column
+    });
+
+    it('should filter the Node table', () => {
+        // filtering by Node name should only display rows with a matching name
+        // filtering by CVE Severity should only display rows with a matching severity
+        // filtering by CVE Status should only display rows with a matching status
+        // filtering by CVSS should only display rows with a CVSS in range
+        // filtering by Cluster should only display rows with a matching cluster
+        // filtering by Operating System should only display rows with a matching OS
+        // filtering by Component should only display rows with a nested table containing a matching component
+        //   - expand each row
+        //   - check that the component name exists in the table
+        // clearing filters should remove all filter chips and filter from the URL
+    });
+
+    // Note: This might not be reliable in CI due to the low number of Nodes. We may need to mock and/or
+    // just test the logic of the pagination in the URL.
+    it('should correctly paginate the Node table', () => {
```

Comment:
If you will forgive a comment from a former editor, **correctly** in the and the following test contrasts with direct and simple `'should filter the Node table'` in the preceding test.

Reply:
Good point, I'll remove this. Also redundant since we don't typically test that features are working **incorrectly** 😄